snappy_pipeline/workflows/ngs_mapping/__init__.py

```diff
@@ -362,28 +362,29 @@
     # Configuration for STAR
     star:
       path_index: REQUIRED # Required if listed in ngs_mapping.tools.rna; otherwise, can be removed.
+      path_features: ""    # Required for computing gene counts
       num_threads_align: 16
       num_threads_trimming: 8
       num_threads_bam_view: 4
       num_threads_bam_sort: 4
       memory_bam_sort: 4G
+      genome_load: NoSharedMemory
+      raw_star_options: ''
+      align_intron_max: 1000000                # ENCODE option
+      align_intron_min: 20                     # ENCODE option
+      align_mates_gap_max: 1000000             # ENCODE option
+      align_sjdb_overhang_min: 1               # ENCODE option
+      align_sj_overhang_min: 8                 # ENCODE option
+      out_filter_mismatch_n_max: 999           # ENCODE option
+      out_filter_mismatch_n_over_l_max: 0.04   # ENCODE option
+      out_filter_multimap_n_max: 20            # ENCODE option
+      out_filter_type: BySJout                 # ENCODE option
+      out_filter_intron_motifs: None    # or for cufflinks: RemoveNoncanonical
+      out_sam_strand_field: None        # or for cufflinks: intronMotif
+      transcriptome: false              # true to output transcript coordinate bam for RSEM
```

Comment:
This can lead to conflicts with `path_features`. As far as I understand Star can only have one parameter for quantmode

Reply:
That's correct, `STAR` parameter `--quantMode` can appear only once in the list of parameters for `STAR`.

That's why the wrapper builds a separate variable (`quant_mode`) which can take no value, or either `GeneCounts` or `TranscritpomeSAM` or both.